src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -260,4 +262,40 @@ public function unfollowAction(Request $request, CitizenProject $citizenProject)
             ],
         ]);
     }
+
+    /**
+     * @Route("", name="app_citizen_project_landing")
+     * @Method("GET")
+     */
+    public function landingAction(): Response
+    {
+        $city = $this->getUser() ? $this->getUser()->getCityName() : SearchParametersFilter::DEFAULT_CITY;
+
+        return $this->render('citizen_project/landing.html.twig', [
+            'city' => $city,
+        ]);
+    }
+
+    /**
+     * @Route("/landing/results", name="app_citizen_project_landing_result", condition="request.isXmlHttpRequest() and request.query.get('city')")
+     * @Method("GET")
+     */
+    public function landingPageResultAction(Request $request, CitizenProjectManager $citizenProjectManager): Response
+    {
+        $search = $this->get(SearchParametersFilter::class);
+        $search->setCity($request->query->get('city'));
+
+        try {
+            $results = $citizenProjectManager->findNearCitizenProjectByCoordinates($search->getCityCoordinates());
+            $citizenProjectManager->injectCitizenProjectCreator($results);
+
+            $response = $this->render('citizen_project/_landing_results.html.twig', [
+                'results' => $results,
+            ]);
+        } catch (GeocodingException $exception) {
+            $response = new Response();
```

Comment:
Réponse 200 ???

Reply:
Should be a 404 with a correct debug message.